Cargo.toml

```diff
@@ -8,13 +8,12 @@ documentation = "https://docs.rs/qingke"
 homepage = "https://github.com/ch32-rs/qingke"
 categories = ["embedded", "no-std", "hardware-support"]
 license = "MIT/Apache-2.0"
-version = "0.2.1" # for rt and macros
+version = "0.3.0" # for rt and macros
 edition = "2021"
 
 [package]
 name = "qingke"
-# version.workspace = true
-version = "0.2.0"
+version.workspace = true
```

Comment:
I think it is reasonable to unify these versions. They are really tightly coupled crates.